### DISCUSSION TITLE
Move the cell-options-menu to the top-right

### DISCUSSION THREAD

**Discussion by krish-adi** [microsoft/vscode-jupyter#10325] [Questions and Answers] [answered]
How do I move the cell-option menu to the top-right of the cell?

Like this:
<img width="1113" alt="Screenshot 2022-06-05 at 17 43 17" src="https://user-images.githubusercontent.com/6900550/172058803-efc38979-450f-4945-a578-b0e934aabfb7.png">

**Comment by DonJayamanne** [comment, marked as answer]
You can move it to the right using the following setting:

<img width="436" alt="Screen Shot 2022-06-06 at 08 50 16" src="https://user-images.githubusercontent.com/1948812/172073856-c0c769ee-85f9-471f-884c-76726b64ca3d.png">

Here's the corresponding setting for the  `settings.json` file
```json
  "notebook.cellToolbarLocation": {
    "default": "right"
  },
```

**Reply by krish-adi** [reply]
Works !
